platforms/s32k1xx/bsp/bspUart/test/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(bspUartTest src/IncludeTest.cpp)
```

Comment:
"add_subdirectory" needs to be added to CMakeLists in OpenBSW root (in the case of BUILD_UNIT_TESTS and S32K1XX).